### PR TITLE
feat(Picker): add new style of Picker to support dividerColor

### PR DESCRIPTION
**Pull request by ZengBeauty**
## Description

Add new style of Picker to support dividerColor in IOS.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by youngjuning** [comment]
1

File: src/components/picker-view/TYRCTPicker.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
+/* eslint-disable */
```
Comment (ShinyLeee) [inline src/components/picker-view/TYRCTPicker.js:1]: 注意下 eslint

File: src/components/picker-view/index.ios.js [hunk 2]
```diff
@@ -92,7 +100,10 @@ export class PickerView extends PureComponent {
 PickerView.Item = PickerIOS.Item;
 
 const ThemedPickerView = props => {
-  const { theme: localTheme, itemStyle, ...rest } = props;
+  const { theme: localTheme, itemStyle, useTYRCTPicker, ...rest } = props;
+  const appRnVersion = get(TYNative, 'mobileInfo.appRnVersion');
```
Comment (ShinyLeee) [inline src/components/picker-view/index.ios.js:104]: 可采用 UIManager 里是否存在该组件判断更为准确

File: src/components/picker-view/picker/index.ios.js [hunk 3]
```diff
@@ -1,3 +1,117 @@
...
+     */
+    textSize: PropTypes.number,
+    /**
+     * 字体宽度
+     */
+    fontWeight: PropTypes.string,
+    /**
+     * 字体
+     */
+    fontFamily: PropTypes.string,
```
Comment (ShinyLeee) [inline src/components/picker-view/picker/index.ios.js:44]: fontFamily 正常应该为有限的集合

File: src/components/date-picker/datePicker.js [hunk 4]
```diff
@@ -511,6 +511,14 @@ class DatePicker extends React.Component {
 
   render() {
     const { value, cols } = this.getIndexAndCols();
+    let pickerFontSize = 30;
```
Comment (ShinyLeee) [inline src/components/date-picker/datePicker.js:514]: 不需要 convertX ？

File: src/components/picker-view/picker/index.ios.js [hunk 5]
```diff
@@ -1,3 +1,117 @@
-import { PickerIOS } from 'react-native';
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { requireNativeComponent, processColor, ViewPropTypes } from 'react-native';
 
-export default PickerIOS;
+const TYRCTPicker = requireNativeComponent('TYRCTPicker');
```
Comment (ShinyLeee) [inline src/components/picker-view/picker/index.ios.js:5]: requireNativeComponent 优化下，避免不支持的 app 环境崩溃

Comment (ShinyLeee) [reply src/components/picker-view/picker/index.ios.js:5]: https://stackoverflow.com/questions/48953434/check-if-native-ui-component-does-exist

File: src/components/popup/picker.js [hunk 6]
```diff
@@ -152,6 +153,14 @@ class PickerPopup extends React.PureComponent {
       ...props
     } = this.props;
     const pickerDatas = singlePicker ? [dataSource] : dataSource;
+    let pickerFontSize = 30;
+    if (pickerDatas.length < 3) {
+      pickerFontSize = 30;
```
Comment (ShinyLeee) [inline src/components/popup/picker.js:158]: convertX ?

File: src/components/theme/base.js [hunk 7]
```diff
@@ -52,17 +52,17 @@ const text = {
  */
 const picker = {
   light: {
-    fontSize: 16,
-    fontColor: '#000',
-    dividerColor: getDividerColor, // 预留 IOS 暂不支持
-    unitFontSize: 16,
-    unitFontColor: '#000',
+    fontSize: 30,
```
Comment (ShinyLeee) [inline src/components/theme/base.js:55]: 是否有兼容问题，如果未启用新版本的 picker 却改大了字体大小

**Review by ShinyLeee — CHANGES_REQUESTED** [review]
评论的上述几个问题留意下

**Review by ShinyLeee — APPROVED** [review]

**Comment by ZengBeauty** [comment]
@ShinyLeee ts typing https://github.com/tuya/DefinitelyTyped/commit/89d2f41f1f3f2f72fa62de2e9741b749b483cdb5

**Review by youngjuning — APPROVED** [review]
本地已验证，没有发现问题。

**Pull request merged**